packages/block-editor/src/components/inserter/menu.js

```diff
@@ -223,8 +202,50 @@ function InserterMenu(
 		setSelectedTab( value );
 	};
 
+	const patternsTab = useMemo(
```

Comment:
It was memoized before: https://github.com/WordPress/gutenberg/pull/60257/files#diff-2745181e57f35c49a91ec773fd2b1740c75f87b0f74efb857dd7a08a90b064e7L158.

I can't remember why I moved it down in the file though. I'll see if I can move it closer to its previous position so that relationship in git is clearer.